certbot/cli.py

```diff
@@ -1190,6 +1203,9 @@ def _paths_parser(helpful):
         help=config_help("work_dir"))
     add("paths", "--logs-dir", default=flag_default("logs_dir"),
         help="Logs directory.")
+    add("paths", "--certbot-dir", default=None,
+        help="Asks Certbot to keep all of its files below a given directory. --certbot-dir $DIR "
+             "is equivalent to --config-dir $DIR --work-dir $DIR/.internal/ --logs-dir $DIR/logs")
```

Comment:
Can you reference `constants.WORK_SUBDIR` and `constants.LOGS_SUBDIR` in the help message to ensure the message is updated if the constants ever change?